diff --git a/build.gradle.kts b/build.gradle.kts
index f51d9f1c..6357f590 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -55,6 +55,7 @@ dependencies {
     implementation(libs.slf4j.jul)
     implementation(libs.logback)
     implementation(project(":libsignal-cli"))
+	implementation(files("/tmp/libsignal-client.jar"))
 }
 
 configurations {
@@ -63,6 +64,10 @@ configurations {
     }
 }
 
+configurations.all {
+	exclude(group = "org.signal", module = "libsignal-client")
+}
+
 
 tasks.withType<AbstractArchiveTask>().configureEach {
     isPreserveFileTimestamps = false
