diff --git a/build.gradle.kts b/build.gradle.kts
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -89,6 +89,7 @@ dependencies {
     implementation(libs.slf4j.jul)
     implementation(libs.logback)
     implementation(libs.zxing)
     implementation(project(":libsignal-cli"))
+	implementation(files("/tmp/libsignal-client.jar"))
 }

@@ -97,6 +98,10 @@ configurations {
     }
 }

+configurations.all {
+	exclude(group = "org.signal", module = "libsignal-client")
+}
+

 tasks.withType<AbstractArchiveTask>().configureEach {
     isPreserveFileTimestamps = false
