diff --git a/build.gradle.kts b/build.gradle.kts
index d8e36ea4..dba4dae3 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -91,6 +91,7 @@ dependencies {
     implementation(libs.logback)
     implementation(libs.zxing)
     implementation(project(":libsignal-cli"))
+	implementation(files("/tmp/libsignal-client.jar")) 
 }
 
 configurations {
@@ -99,6 +100,9 @@ configurations {
     }
 }
 
+configurations.all {
+    exclude(group = "org.signal", module = "libsignal-client")
+}
 
 tasks.withType<AbstractArchiveTask>().configureEach {
     isPreserveFileTimestamps = false
